.github/workflows/finishReleaseCycle.yml

```diff
@@ -6,12 +6,38 @@ on:
 
 # The updateProduction and createNewStagingDeployCash jobs are executed when a StagingDeployCash is closed.
 jobs:
+  checkDeployBlockers:
+    runs-on: ubuntu-latest
+    if: contains(github.event.issue.labels.*.name, 'StagingDeployCash')
+
+    outputs:
+      hasDeployBlockers: ${{ steps.checkDeployBlockers.outputs.HAS_DEPLOY_BLOCKERS }}
+
+    steps:
+      - name: Check for any deploy blockers
+        id: checkDeployBlockers
+        uses: Expensify/Expensify.cash/.github/actions/checkDeployBlockers@master
+        with:
+          GITHUB_TOKEN: ${{ secrets.OS_BOTIFY_TOKEN }}
+          ISSUE_NUMBER: ${{ github.events.issue.number }}
```

Comment:
Are you sure it's `github.events` and not `github.event` ?

Reply:
Nope. That was a typo.